### ISSUE TITLE
[HOLD #21766][Guides] Notifications are showing for chats that won't appear

### ISSUE THREAD

**Issue by kevinksullivan** [Expensify/App#22600]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Logged out of Newdot
2. Logged back in
3. Change view preference from "Most Recent" to "Focus Mode" 

## Expected Result:
Only show notifications for chats that are unread

## Actual Result:
On the tab at the top I am seeing (10) new messages but no new messages are appearing

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [X] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [X] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** v1.3.33-4
**Reproducible in staging?:** Y
**Reproducible in production?:** Y

**Email or phone of affected tester (no customers):** madison.grumbles@team.expensify.com
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

[Screen recording 2023-06-28 5.05.02 PM.webm](https://github.com/Expensify/App/assets/22508290/6947d450-6b0b-48ca-9031-6ea5a1b65bf4)

![image](https://github.com/Expensify/App/assets/22508290/8dad57d7-ebbe-435d-9894-bc8fb45f96de)

**Issue reported by:** Guide
**Slack conversation:** https://expensify.slack.com/archives/C036S0BVAHH/p1687990570638379

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~012f58dac5ddba9c49</li>
        <li>Upwork Job ID: 1678559511194476544</li>
        <li>Last Price Increase: 2023-07-11</li>
    </ul>
</details>

**Comment by kevinksullivan** [comment]
Note that this may related to https://github.com/Expensify/App/issues/21766 CC @marcaaron

**Comment by flodnv** [comment]
HOLDing on https://github.com/Expensify/App/issues/21766 for now based on @marcaaron assessment: https://expensify.slack.com/archives/C04JD99283A/p1689099523743349?thread_ts=1689041354.773319&cid=C04JD99283A

**Comment by flodnv** [comment]
HOLDing on https://github.com/Expensify/App/issues/21766 for now based on @marcaaron assessment: https://expensify.slack.com/archives/C04JD99283A/p1689099523743349?thread_ts=1689041354.773319&cid=C04JD99283A

**Comment by flodnv** [comment]
Still on HOLD.

**Comment by sonialiap** [comment]
Still on hold

**Comment by sonialiap** [comment]
This issue is pretty old so the reported behavior may no longer be the same. As per [this thread](https://expensify.slack.com/archives/C04JD99283A/p1696296332416169) we have a number of issues related to this. One, 
https://github.com/Expensify/App/issues/27512 has a PR out for review. I'm going to close this issue. If the reported behavior is still present after #27512 is on dev then please open a new issue with updated reproduction steps.
